Entity/Concrete/ProductDetails.cs

```diff
@@ -1,34 +1,32 @@
-﻿using System;
-using WebNetSample.Core.Entities;
-using WebNetSample.Entity.Concrete;
+﻿using WebNetSample.Core.Entities;
 
-namespace Entity.Concrete
+namespace WebNetSample.Entity.Concrete;
+
+public class ProductDetails : BaseEntity
 {
-    public class ProductDetails : BaseEntity
-    {
-        public string Name { get; set; }
+    public string Name { get; set; }
 
-        public decimal Price { get; set; }
+    public decimal Price { get; set; }
 
-        public string ImageUrl { get; set; }
+    public string ImageUrl { get; set; }
 
-        public string CategoryName { get; set; }
+    public string CategoryName { get; set; }
 
-        public string SupplierName { get; set; }
+    public string SupplierName { get; set; }
 
-        public ProductDetails()
-        {
-        }
+    public ProductDetails()
+    {
+    }
 
-        public ProductDetails(Guid id, string categoryName, string supplierName, string name,
-            decimal price, string imageUrl) : this()
-        {
-            Id = id;
-            CategoryName = categoryName;
-            SupplierName = supplierName;
-            Name = name;
-            Price = price;
-            ImageUrl = imageUrl;
-        }
+    public ProductDetails(Guid id, string categoryName, 
```

Comment:
Well, meaning wrapping of args into a column I was meaning this example:
```suggestion
    public ProductDetails(
           Guid id,
           string categoryName, 
           string supplierName, 
           string name,
           ...
```

Each argument should have a separate row